tools/python/boutiques/bosh.py

```diff
@@ -423,6 +396,13 @@ def runs_as_cli():
         else:
             parser.print_help()
             raise SystemExit
+    except ZenodoError as e:
+        # We don't want to raise an exception when function is called
+        # from CLI.'
+        if runs_as_cli():
+            print(e.message)
```

Comment:
Running locally I had the following error:
```
$ bosh publish ./boutiques/schema/examples/example1/example1.json "Test GK" "Test McGill" --sandbox -v --zenodo-token hAaW2wSBZMskxpfigTYHcuDrCPWr2VeQZgBLErKbfF5RdrKhzzJi8i2hnN8r -y
[ INFO ] Using Zenodo endpoint https://sandbox.zenodo.org
[ INFO (401) ] Zenodo is accessible
[ INFO (200) ] Authentication to Zenodo successful
[ INFO (201) ] Deposition succeeded, id is 213236

Traceback (most recent call last):
  File "/Users/gkiar/code/gkiar/boutiques/tools/python/boutiques/bosh.py", line 385, in bosh
    out = publish(*params)
  File "/Users/gkiar/code/gkiar/boutiques/tools/python/boutiques/bosh.py", line 233, in publish
    results.zenodo_token).publish()
  File "/Users/gkiar/code/gkiar/boutiques/tools/python/boutiques/publisher.py", line 201, in publish
    self.zenodo_upload_descriptor(deposition_id)
  File "/Users/gkiar/code/gkiar/boutiques/tools/python/boutiques/publisher.py", line 167, in zenodo_upload_descriptor
    self.raise_zenodo_error("Cannot upload descriptor", r)
  File "/Users/gkiar/code/gkiar/boutiques/tools/python/boutiques/publisher.py", line 184, in raise_zenodo_error
    .format(r.status_code, message))
boutiques.publisher.ZenodoError: Zenodo error (504): Cannot upload descriptor.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/gkiar/code/gkiar/boutiques/tools/python/env/boutiques3/bin/bosh", line 11, in <module>
    load_entry_point('boutiques', 'console_scripts', 'bosh')()
  File "/Users/gkiar/code/gkiar/boutiques/tools/python/boutiques/bosh.py", line 403, in bosh
    print(e.message)
AttributeError: 'ZenodoError' object has no attribute 'message'
```

This can be fixed by replacing `e.message` with `e`, based on how you defined the exception.